contrib/grinder/properties/grinder.properties

```diff
@@ -1,18 +1,21 @@
-grinder.script=../scripts/tests.py
```

Comment:
this file is meant for tests.  i like the idea of creating another one for localhost, but that should be a separate properties file, (and it should be mentioned in the readme.)